README.md

```
@@ -178,3 +178,58 @@ Apache 2.0 - see [LICENSE](LICENSE).
     <img alt="Star History Chart" src="https://api.star-history.com/svg?repos=aegra/aegra&type=Date" />
   </picture>
 </a>
+
+## ❓ FAQ
+
+### What is Aegra?
+Aegra is a drop-in replacement for LangSmith Deployments. Use the same LangGraph SDK and APIs, but run it on your own infrastructure with PostgreSQL persistence.
+
+### Key Features
+| Feature | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)